AppController/djinn.rb

```diff
@@ -5743,6 +5743,7 @@ def try_to_scale_down(app_name, delta_appservers)
           @last_decision[app_name] = Time.now.to_i
           Djinn.log_info("Removing an AppServer for #{app_name} #{location}.")
           num_to_remove -= 1
+          examine_scaling_down_instances(app_name, node_ip)
```

Comment:
No it shouldn't, so above in the downscaling of AppServers (https://github.com/AppScale/appscale/blob/master/AppController/djinn.rb#L5738) it gets just a list of appengine nodes and goes through them in reverse order. The scaling down of instances, grabs the same node ip for which the last AppServer was removed and checks if there are any more running.